components/03-modules/filters/Filters.js

```diff
@@ -1,8 +1,10 @@
+import uniqueId from 'lodash.uniqueid'
```

Comment:
do we need uniqueId?

Reply:
I remove it, after changes we don't need it.